### PR TITLE
Add Orbit-X Keyboard

### PR DESCRIPTION
**Pull request by ai03-2725**
## Description

Adds support for the Orbit-X keyboard and its PCB.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* Adds support for Orbit-X

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/ai03/orbit_x/readme.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Orbit-X
+
+![orbit_x](https://i.imgur.com/4z2wgsV.png)
+
+A compact high-end split keyboard  
+
+* Keyboard Maintainer: [ai03](https://github.com/ai03-2725)  
+* Hardware Supported: The Orbit-X keyboard PCB  
+* Hardware Availability: Deskeys for official kit; open-source PCB [here](https://github.com/ai03-2725/Orbit/tree/master/OrbitX-PCB)
```
Comment (fauxpark) [inline keyboards/ai03/orbit_x/readme.md:9]:
```suggestion
* Keyboard Maintainer: [ai03](https://github.com/ai03-2725)
* Hardware Supported: The Orbit-X keyboard PCB
* Hardware Availability: Deskeys for official kit; open-source PCB [here](https://github.com/ai03-2725/Orbit/tree/master/OrbitX-PCB)
```

File: keyboards/ai03/orbit_x/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+}
+
+void matrix_scan_user(void) {
+
+}
+
+bool led_update_user(led_t led_state) {
+    return true;
+}
+*/
```
Comment (fauxpark) [inline keyboards/ai03/orbit_x/keymaps/default/keymap.c:73]:
```suggestion
```

File: keyboards/ai03/orbit_x/info.json [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+                {"x":2, "y":3.13},
+                {"x":10.5, "y":3.13},
+                {"x":5.25, "y":3.25, "h":1.5},
+                {"x":7.25, "y":3.25, "h":1.5},
+                {"x":0, "y":3.38},
+                {"x":1, "y":3.38},
+                {"x":4.25, "y":3.38},
+                {"x":8.25, "y":3.38},
+                {"x":11.5, "y":3.38},
+                {"x":12.5, "y":3.38}
```
Comment (noroadsleft) [inline keyboards/ai03/orbit_x/info.json:57]: The KLE to info.json Converter doesn't play nice with keyboards that run column stagger.

```suggestion
                {"x":0, "y":0.38},
                {"x":1, "y":0.38},
                {"x":2, "y":0.13},
                {"x":3, "y":0},
                {"x":4, "y":0.13},
                {"x":5, "y":0.25},
                {"x":7.5, "y":0.25},
                {"x":8.5, "y":0.13},
                {"x":9.5, "y":0},
                {"x":10.5, "y":0.13},
                {"x":11.5, "y":0.38},
                {"x":12.5, "y":0.38},
                {"x":0, "y":1.38},
                {"x":1, "y":1.38},
                {"x":2, "y":1.13},
                {"x":3, "y":1.0},
                {"x":4, "y":1.13},
                {"x":5, "y":1.25},
                {"x":7.5, "y":1.25},
                {"x":8.5, "y":1.13},
                {"x":9.5, "y":1.0},
                {"x":10.5, "y":1.13},
                {"x":11.5, "y":1.38},
                {"x":12.5, "y":1.38},
                {"x":0, "y":2.38},
                {"x":1, "y":2.38},
                {"x":2, "y":2.13},
                {"x":3, "y":2},
                {"x":4, "y":2.13},
                {"x":5, "y":2.25},
                {"x":7.5, "y":2.25},
                {"x":8.5, "y":2.13},
                {"x":9.5, "y":2},
                {"x":10.5, "y":2.13},
                {"x":11.5, "y":2.38},
                {"x":12.5, "y":2.38},
                {"x":0, "y":3.38},
                {"x":1, "y":3.38},
                {"x":2, "y":3.13},
                {"x":3, "y":3},
                {"x":4.25, "y":3.38},
                {"x":5.25, "y":3.25, "h":1.5},
                {"x":7.25, "y":3.25, "h":1.5},
                {"x":8.25, "y":3.38},
                {"x":9.5, "y":3},
                {"x":10.5, "y":3.13},
                {"x":11.5, "y":3.38},
                {"x":12.5, "y":3.38}
```

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**